src/property-filter/filtering-token/styles.scss

```diff
@@ -8,13 +8,32 @@
 @use '../../internal/styles/tokens' as awsui;
 @use '../../token-group/constants' as constants;
 
-.root {
+$token-padding-block: styles.$control-padding-vertical;
+$token-padding-inline: styles.$control-padding-horizontal;
+$token-grouped-padding-block: 2px;
+$token-grouped-padding-inline: styles.$control-padding-horizontal;
+$inner-token-padding-block: 1px;
+$inner-token-padding-inline: styles.$control-padding-horizontal;
+
+.root,
+.inner-root {
+  @include styles.styles-reset;
+
   display: flex;
   align-content: stretch;
+
+  // Make tokens larger so that individual and group tokens appear of the same size.
+  &.has-groups {
+    min-block-size: calc(2px + #{awsui.$size-vertical-input});
```

Comment:
is there any way to introduce a token instead of the 'magic number' for this value instead?

Reply:
I see no value in having a token here at this point because the adjustment does not need to be configurable and there is no token which can account for possible future changes of the spacing that might affect the formula.

In either case, this particular style might get updated after a VR review. At the moment it is not available publicly as the property filter is not yet updated to use the grouped tokens.

@katiegeorge , what do you think?